src/lib/helpers.ts

```diff
@@ -245,9 +245,13 @@ export function generatePKCEVerifier() {
   const verifierLength = 56
   const array = new Uint32Array(verifierLength)
   if (typeof crypto === 'undefined') {
-    throw new Error(
-      'PKCE is not supported on devices without WebCrypto API support, please add polyfills'
-    )
+    const charSet = 'ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789'
```

Comment:
/nit Consider adding the [four punctuation characters](https://www.oauth.com/oauth2-servers/pkce/authorization-request/)
```suggestion
    const charSet = 'ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789-._~'
```